runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -28,7 +28,9 @@ namespace misc {
     Interval();
     explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
     Interval(ssize_t a_, ssize_t b_);
-    virtual ~Interval() {};
+    Interval(Interval const&) = default;
+    virtual ~Interval();
```

Comment:
Code formatting: d-tors after c-tors, but before rest of declarations/definitions.